src/layouts/Folders/components/MenuDropdownButton.tsx

```diff
@@ -32,7 +32,10 @@ export const MenuDropdownButton = (): JSX.Element => {
           >
             Create page
           </Button>
-          <ContextMenu>
+          <ContextMenu
+            placement="bottom-end"
+            offset={[0, 0]} // to prevent weird gap between the dropdown menu and button>
```

Comment:
could we give a brief overview of what `offset` does (eg: this positions it (x, y) units away relative from the base position)?

Reply:
added in a comment adc5ecf